uktrade/data-hub-frontend
# ReferralList component

## Description of change

Adds the `ReferralList` component, which lists all referrals related to the logged in user.

## Test instructions

The component is not used anywhere yet, so the only way to see it is to check out this branch and put the `ReferallList` component somewhere.

## Screenshots
<img width="775" alt="Screen Shot 2020-02-28 at 2 08 44 PM" src="https://user-images.githubusercontent.com/2333157/75555308-e630a100-5a33-11ea-92fe-bfc46ffea13b.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
